src/js/post-911-gib-status/reducers/index.js

```diff
@@ -1,19 +1,33 @@
 import set from 'lodash/fp/set';
 
+import {
+  BACKEND_AUTHENTICATION_ERROR,
+  BACKEND_SERVICE_ERROR,
+  GET_ENROLLMENT_DATA_FAILURE,
+  GET_ENROLLMENT_DATA_SUCCESS,
+  NO_CHAPTER33_RECORD_AVAILABLE
+} from '../utils/constants';
+
 const initialState = {
   enrollmentData: null,
   availability: 'awaitingResponse'
 };
 
 function post911GIBStatus(state = initialState, action) {
   switch (action.type) {
-    case 'GET_ENROLLMENT_DATA_SUCCESS':
+    case GET_ENROLLMENT_DATA_SUCCESS:
       return {
         ...state,
         enrollmentData: action.data,
         availability: 'available'
       };
-    case 'GET_ENROLLMENT_DATA_FAILURE':
+    case BACKEND_SERVICE_ERROR:
+      return set('availability', 'backendServiceError', state);
+    case BACKEND_AUTHENTICATION_ERROR:
+      return set('availability', 'backendAuthenticationError', state);
+    case NO_CHAPTER33_RECORD_AVAILABLE:
+      return set('availability', 'noChapter33Record', state);
+    case GET_ENROLLMENT_DATA_FAILURE:
```

Comment:
I think I answered this in the above question, but this is the catch-all for any failure not recognized/expected. I have no idea when/why/if this would happen.